package.json

```diff
@@ -188,6 +188,11 @@
           "description": "Enable/disable autoclosing of XML tags. \n\nIMPORTANT: Turn off editor.autoClosingTags for this to work",
           "scope": "window"
         },
+        "xml.codeLens.enabled": {
+          "type": "boolean",
+          "default": false,
+          "description": "Enable/disable the references CodeLens"
```

Comment:
Change references CodeLens to XML Codelens because we can have other CodeLens features since now you have an extension for that